test/fixtures/draft_order_completed.json

```diff
@@ -0,0 +1,159 @@
+{
+  "draft_order": {
+    "id": 517119332,
+    "note": "This is a note",
+    "email": "montana_hilpert@example.com",
+    "taxes_included": false,
+    "currency": "CAD",
+    "subtotal_price": "1007.41",
+    "total_tax": "0.00",
+    "total_price": "1027.41",
+    "invoice_sent_at": null,
+    "created_at": "2017-02-02T13:14:38-05:00",
+    "updated_at": "2017-02-02T13:14:38-05:00",
+    "tax_exempt": false,
+    "completed_at": null,
```

Comment:
This shouldn't be null; and should also be asserted in the tests above. This code gets used as a reference so we should reflect reality here.